main.py

```diff
@@ -30,7 +31,36 @@ def Remove(self):
     def Add(self):
         if self._Rear < self._MaxSize - 1:
             TileLibrary = CreateTileLibrary()
-            RandNo = random.randint(0, 26)
+            tile_frequency = [
```

Comment:
Can this be shortened into a single line, even if it runs over a bit? Or at least move the closing brace up a line.